### PR TITLE
Add jquery-touch-events with git auto-update

### LINKED ISSUE #7968: [Request] Add Jquery-Touch-Events
**Library name:**  Jquery-Touch-Events
**Git repository url:** https://github.com/benmajor/jQuery-Touch-Events
**npm package url(optional):** git+https://github.com/benmajor/jQuery-Touch-Events.git
**License(s):** MIT 
**Official homepage:** https://github.com/benmajor/jQuery-Touch-Events
**Wanna say something? Leave message here:**

This package is super useful, has a decent following, and is the best lightweight abstraction of unifying touch events for jQuery that I have come across. 

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by x09326**
PR for #7968 
@dannyxx001 please help me review it, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/benmajor/jQuery-Touch-Events
 * Official website (optional, not the repository):
 * NPM package url (optional):https://www.npmjs.com/package/benmajor-jquery-touch-events
 * GitHub / Bitbucket popularity (required):
   - Count of stars:477
   - Count of watchers:39
   - Count of forks:183
 * NPM download stats (optional):
   - Downloads in the last day:0
   - Downloads in the last week:7
   - Downloads in the last month:26

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7968, cc @benmajor

### CODE REVIEW

**Comment by dannyxx001** [comment]
I think this PR is OK. @PeterDaveHello please help us review again, thanks.

**Comment by PeterDaveHello** [comment]
The npm package is not maintained by its author, so we are not going to use it.

BTW, there are all the versions here: https://github.com/benmajor/jQuery-Touch-Events/tree/master/src, but not all tagged, @x09326 maybe you can help us manually add them all.

**Comment by x09326** [comment]
@PeterDaveHello 
So I need to use git auto-update, and manually add all of versions.
Right?

**Comment by PeterDaveHello** [comment]
Yes.

**Comment by x09326** [comment]
@dannyxx001 
I have changed to git auto-update, and added all versions.
Please help me review it, thanks.

**Comment by dannyxx001** [comment]
I think the PR is OK. @PeterDaveHello please help us review again, thanks.

**Comment by PeterDaveHello** [comment]
This PR doesn't pass the test yet. So I won't review it.

**Comment by PeterDaveHello** [comment]
@dannyxx001 Please ping me after it passed the test. It doesn't need to be so hurry.

**Comment by dannyxx001** [comment]
ping @PeterDaveHello

File: ajax/libs/jquery-touch-events/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "jquery-mobile",
+    "mobile",
+    "touch"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/benmajor/jQuery-Touch-Events.git",
+    "basePath": "",
+    "files": [
+      "src/jquery.mobile-events*"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-touch-events/package.json]: `src` should be base path

**Comment by RyKilleen** [comment]
Thank you all for your work! Super appreciated.

**Comment by PeterDaveHello** [comment]
@RyKilleen No probs.

**Pull request merged**